docs/useStorage.md

```diff
@@ -1,46 +1,51 @@
+
 # useStorage Hook
 
  The react useStorage hook is a react hook which allows use to easily manage state in a web storage API like a localstorage or sessionstorage.<br>
```

Comment:
It's not a react hook. It is a hook written in react, but it's not coming from react